src/main/java/com/easyrest/components/ActionOnRestaurants.java

```diff
@@ -0,0 +1,87 @@
+package com.easyrest.components;
+
+import com.easyrest.pages.moderatorPanel.*;
+import com.easyrest.pages.OwnerPanel;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class ActionOnRestaurants {
```

Comment:
I think we should not add this class now, because if we add we will have class that duplicate another class. I rename my class ActionOnUser and you can use it for your role (i need task for it).

Reply:
I have a class called- ActionOnRestaurants
your class is called -ActionOnUser
where is the duplication?